app/src/main/java/org/p2p/wallet/swap/ui/jupiter/main/widget/SwapWidgetModel.kt

```diff
@@ -0,0 +1,54 @@
+package org.p2p.wallet.swap.ui.jupiter.main.widget
+
+import org.p2p.core.common.TextContainer
+import org.p2p.uikit.utils.skeleton.SkeletonCellModel
+import org.p2p.uikit.utils.text.TextViewCellModel
+import org.p2p.uikit.utils.toPx
+import org.p2p.wallet.R
+
+sealed interface SwapWidgetModel {
+    data class Content(
+        val isStatic: Boolean = false,
+        val widgetTitle: TextViewCellModel? = null,
+        val availableAmount: TextViewCellModel? = null,
+        val currencyName: TextViewCellModel? = null,
+        val amount: TextViewCellModel? = null,
+        val balance: TextViewCellModel? = null,
+        val fiatAmount: TextViewCellModel? = null,
+    ) : SwapWidgetModel
+
+    data class Loading(
+        val isStatic: Boolean = false,
+        val widgetTitle: TextViewCellModel.Raw,
+        val currencySkeleton: TextViewCellModel.Skeleton = TextViewCellModel.Skeleton(bigSkeleton()),
+        val amountSkeleton: TextViewCellModel.Skeleton = TextViewCellModel.Skeleton(bigSkeleton()),
+        val balanceSkeleton: TextViewCellModel.Skeleton = TextViewCellModel.Skeleton(smallSkeleton()),
+    ) : SwapWidgetModel
+}
+
+fun swapToAmountLoading(
```

Comment:
better place as a method for  Content class to remove ambiguity in autocomplete and search